scripts/create_elmo_embeddings_from_vocab.py

```diff
@@ -0,0 +1,87 @@
+# pylint: disable=no-self-use
+
+import os
+import argparse
+import sys
+
+sys.path.insert(0, os.path.dirname(os.path.abspath(os.path.join(__file__, os.pardir))))
+
+import h5py
+import numpy
+import torch
+from torch.autograd import Variable
+
+from allennlp.data.token_indexers import ELMoTokenCharactersIndexer
+from allennlp.modules.elmo import _ElmoCharacterEncoder
+from allennlp.data import Token, Vocabulary
+
+
+def main(vocab_path: str,
```

Comment:
Can you move this to somewhere inside the allennlp/ directory structure so that it can be imported and re-used elsewhere?

Reply:
where do we need to use it inside the library?

Reply:
It'd be nice to reuse in a test case for another PR I'm working on.  But if it's a hassle then I can work around.